### PR TITLE
Improve reference links

### PR DESCRIPTION
**Pull request by yuyu2172**
`~` shortens a symbol by skipping import space information.
This simplifies documentation, but sometimes import information makes the doc easier to understand.

I propose to use shorter notation when a reference is linking a symbol located in a similar import space.

EDIT:
Changed to always use shorter notation.

### CODE REVIEW

**Comment by sanyuwen** [comment]
hello!
yuyu2172
I am a newcomer to deep learning and chainer.
I found it is awesome.
when I want to use your pre-trained model trained by imagenet
model = FasterRCNNVGG16(n_fg_class=len(CLASSES), pretrained_model="imagenet")
it is very slow to download the parameters in the script.
so could you please tell me where I can download the parameters directly? some website?
or can you send me the parameters to my mail?
chuanyi_huang@163.com

**Comment by yuyu2172** [comment]
Weights are found here.
https://github.com/yuyu2172/share-weights/releases

**Comment by sanyuwen** [comment]
thanks 
but I found this error! there is my code.
model = FasterRCNNVGG16(n_fg_class=22,
pretrained_model="faster_rcnn_vgg16_voc07_trained_2017_08_06.npz")
I use the model downloaded from 
https://github.com/yuyu2172/share-weights/releases/download/0.0.4/faster_rcnn_vgg16_voc07_trained_2017_08_06.npz
Which is the corresponding model of the FasterRCNNVGG16.
Traceback (most recent call last):
  File "fasterRCNNforward1.py", line 29, in <module>
    main(args)
  File "fasterRCNNforward1.py", line 21, in main
    model = FasterRCNNVGG16(n_fg_class=22, pretrained_model="faster_rcnn_vgg16_voc07_trained_2017_08_06.npz")
  File "F:\tensorflow\tensorflow\lib\site-packages\chainercv\links\model\faster_rcnn\faster_rcnn_vgg.py", line 147, in __init__
    chainer.serializers.load_npz(pretrained_model, self)
  File "F:\tensorflow\tensorflow\lib\site-packages\chainer\serializers\npz.py", line 140, in load_npz
    d.load(obj)
  File "F:\tensorflow\tensorflow\lib\site-packages\chainer\serializer.py", line 83, in load
    obj.serialize(self)
  File "F:\tensorflow\tensorflow\lib\site-packages\chainer\link.py", line 795, in serialize
    d[name].serialize(serializer[name])
  File "F:\tensorflow\tensorflow\lib\site-packages\chainer\link.py", line 795, in serialize
    d[name].serialize(serializer[name])
  File "F:\tensorflow\tensorflow\lib\site-packages\chainer\link.py", line 551, in serialize
    data = serializer(name, param.data)
  File "F:\tensorflow\tensorflow\lib\site-packages\chainer\serializers\npz.py", line 113, in __call__
    numpy.copyto(value, dataset)
ValueError: could not broadcast input array from shape (84,4096) into shape (92,4096)

what the n_fg_class parameter when you train the model?
How to use the model correctly?

**Comment by yuyu2172** [comment]
Set `n_fg_class` to 20.

Can you discuss this issue here? https://github.com/chainer/chainercv/issues/416
This discussion is irrelevant to this PR.

**Comment by yuyu2172** [comment]
@Hakuyume 
Can you check all documentation PRs?

**Comment by Hakuyume** [comment]
@yuyu2172 OK. I will check them.

File: chainercv/links/model/faster_rcnn/region_proposal_network.py [hunk 1]
```diff
@@ -39,10 +39,10 @@ class agnostic bounding boxes around "objects".
             initialize weight.
             May also be a callable that takes an array and edits its values.
         proposal_creator_params (dict): Key valued paramters for
-            :obj:`chainercv.links.model.faster_rcnn.ProposalCreator`.
+            :obj:`~chainercv.links.model.faster_rcnn.ProposalCreator`.
```
Comment (Hakuyume) [inline chainercv/links/model/faster_rcnn/region_proposal_network.py]: `ProposalCreator` is a class.

**Pull request merged**